### PR TITLE
Update sln configurations

### PR DESCRIPTION
**Pull request by ViktorHofer**
Reacting to the test configuration changes

### CODE REVIEW

**Comment by ViktorHofer** [comment]
@stevenbrix I couldn't repro the issue that you are hitting locally BUT I found out that the configurations in the sln's are outdated. Please give it another try after this and https://github.com/dotnet/corefx/pull/38697 is merged.

File: src/System.Net.Http.WinHttpHandler/System.Net.Http.WinHttpHandler.sln [hunk 1]
```diff
@@ -31,10 +31,10 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{17D5CC82-F72C-4DD2-B6DB-DE7FB2F19C34}.Debug|Any CPU.ActiveCfg = netstandard-Windows_NT-Debug|Any CPU
-		{17D5CC82-F72C-4DD2-B6DB-DE7FB2F19C34}.Debug|Any CPU.Build.0 = netstandard-Windows_NT-Debug|Any CPU
-		{17D5CC82-F72C-4DD2-B6DB-DE7FB2F19C34}.Release|Any CPU.ActiveCfg = netstandard-Windows_NT-Release|Any CPU
-		{17D5CC82-F72C-4DD2-B6DB-DE7FB2F19C34}.Release|Any CPU.Build.0 = netstandard-Windows_NT-Release|Any CPU
+		{17D5CC82-F72C-4DD2-B6DB-DE7FB2F19C34}.Debug|Any CPU.ActiveCfg = netcoreapp-Windows_NT-Debug|Any CPU
```
Comment (davidsh) [inline src/System.Net.Http.WinHttpHandler/System.Net.Http.WinHttpHandler.sln]: Aren't these 'netcoreapp-Windows_NT-*' duplicates of the ones below....the only difference being the GUIDs are different?  Why have two of the same thing now?

Comment (ViktorHofer) [reply src/System.Net.Http.WinHttpHandler/System.Net.Http.WinHttpHandler.sln]: Isn't that the test project? I did use the `UpdateVSConfigurations` target to produce these changes.

Comment (davidsh) [reply src/System.Net.Http.WinHttpHandler/System.Net.Http.WinHttpHandler.sln]: Yes, you're right.  It is the test projects.

**Comment by ViktorHofer** [comment]
Merging as these code paths aren't covered by CI.

**Comment by stevenbrix** [comment]
thanks @ViktorHofer! works like a charm 🙌

**Pull request merged**